### DISCUSSION TITLE
Error read RTMP frame_mbs_only_flag

### DISCUSSION THREAD

**Discussion by lozo2010** [bluenviron/mediamtx#2258] [General]
Hi.

I have a camera with which I stream via RTMP to the server. When I want to consume that stream via RTMP I get the error frame_mbs_only_flag = 0 is not supported.

If I consume it by RTSP I have no problem. Is it possible that there is an option to make it work also for RTMP?

**Comment by aler9** [comment]
Hello, open an Issue and provide a network dump of the traffic between the server and the camera in pcap format (instructions are displayed when you open an issue).

[a comment by an automated account is not shown]
